flamedisx/block_source.py

```diff
@@ -144,6 +147,117 @@ def annotate(self, d: pd.DataFrame):
                           <= d[f'{dim}_max'].values), \
                 f"_annotate of {self} set misordered bounds"
 
+    def bayes_bounds_binomial(self, df, in_dim, supports, rvs_binom, ns_binom, ps_binom, bound):
```

Comment:
I think the bayes_bounds_* functionswould be best as standalone functions in a separate file, rather than among the infrastructure in block_source.py. If you give bounds_prob as an extra argument, the functions won't depend on `self` anymore. Apart from decluttering block_source.py, this will make it easier to unit test the bayes bound functions, and compare them directly to the new implementations in terms of compute later.

Reply:
Done